bigtable/google/cloud/bigtable/instance.py

```diff
@@ -160,10 +171,22 @@ def create(self):
         :returns: The long-running operation corresponding to the create
                     operation.
         """
+        clusters = {}
+        cluster_id = '{}-cluster'.format(self.instance_id)
+        cluster_name = self._client._instance_admin_client.cluster_path(
+            self._client.project, self.instance_id, cluster_id)
+        location = self._client._instance_admin_client.location_path(
+            self._client.project, self._cluster_location_id)
+        cluster = instance_pb2.Cluster(name=cluster_name, location=location,
+                                       serve_nodes=DEFAULT_SERVE_NODES)
```

Comment:
Do we need to expose a way for the caller to set `serve_nodes` explicitly?  What about `default_storage_type`?